### ISSUE TITLE
Hi, any plan about semi-supervised object detction for mmyolo?

### ISSUE THREAD

**Issue by BowieHsu** [open-mmlab/mmyolo#602]
### What is the problem this feature will solve?

semi-supervised object detection for mmyolo.

### What is the feature you are proposing to solve the problem?

Hello Open MMLab team, thank you for deciding to contribute a clean implementation of the YOLO series of algorithms to the community. We have here re-implemented a version of the training code based on YOLOv5 that can support training for YOLOv5/YOLO X/YOLOv7/YOLOv8, and we have also added a semi-supervised training scheme for YOLOv5, which works quite well. Would you be interested in incorporating our semi-supervised scheme into your existing MMYOLO? We would be happy to provide assistance: https://github.com/AlibabaResearch/efficientteacher.

**Comment by hhaAndroid** [comment]
@BowieHsu This is a very practical and important direction. We will research and support it. If you are interested in participating, please feel free to contribute!

**Comment by BowieHsu** [comment]
@hhaAndroid  No problem, you can take a look at our paper first. There aren't too many technical difficulties involved. When designing this semi-supervised scheme, we decided not to modify any network structures of the standard YOLOv5. In fact, what we need is (1) a semi-supervised dataloader, which I have seen has already been implemented in the latest version of mmdetection; (2) a loss function for pseudo-labeling, which we have already written; (3) as well as a mechanism for selecting pseudo-labels and providing auxiliary log information. That's roughly it. I am currently perfecting our official repository and writing a Chinese README. If you have any plans or arrangements, feel free to email me.

**Comment by hhaAndroid** [comment]
@BowieHsu OK. Thank you very much for your reply.
